fail download mklml

使用最新代码 commit https://github.com/PaddlePaddle/Paddle/commit/31632a694c718ac31b890b1b46788f9d70d570c8 进行编译，发现mklml包下载不了。
```
[  3%] Performing download step for 'extern_mklml'

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
make[2]: *** [third_party/mklml/src/extern_mklml-stamp/extern_mklml-download] Error 2
make[1]: *** [CMakeFiles/extern_mklml.dir/all] Error 2
make: *** [all] Error 2
```
使用[mklml.cmake](https://github.com/PaddlePaddle/Paddle/blob/develop/cmake/external/mklml.cmake#L57)中的DOWNLOAD_COMMAND（`wget --no-check-certificate -qO- ${MKLML_URL}`）手动下载，并去掉-q开关：
```
wget --no-check-certificate -O https://github.com/01org/mkl-dnn/releases/download/v0.10/mklml_lnx_2018.0.20170720.tgz
```
错误是`File name too long`，即输出的文件名过长

因此改成`wget --no-check-certificate ${MKLML_URL} -c -O ${MKLML_VER}.tgz`即可。
- 由于这个包比较特殊，建议去掉`-q`打开log日志来观察下载情况 @wangkuiyi 
- 建议加上`-c`断点续传功能，以防突然断网。

--- a/cmake/external/mklml.cmake
+++ b/cmake/external/mklml.cmake
@@ -54,8 +54,8 @@ ExternalProject_Add(
     ${EXTERNAL_PROJECT_LOG_ARGS}
     PREFIX                ${MKLML_SOURCE_DIR}
     DOWNLOAD_DIR          ${MKLML_DOWNLOAD_DIR}
-    DOWNLOAD_COMMAND      wget --no-check-certificate ${MKLML_URL} -c -O ${MKLML_VER}.tgz 
-                          && tar zxf ${MKLML_DOWNLOAD_DIR}/${MKLML_VER}.tgz
+    DOWNLOAD_COMMAND      wget --no-check-certificate ${MKLML_URL} -c -q -O ${MKLML_VER}.tgz 
+                          && tar zxf ${MKLML_VER}.tgz
     DOWNLOAD_NO_PROGRESS  1
     UPDATE_COMMAND        ""
     CMAKE_ARGS            -DCMAKE_INSTALL_PREFIX=${MKLML_INSTALL_ROOT}

--- a/cmake/external/mklml.cmake
+++ b/cmake/external/mklml.cmake
@@ -54,7 +54,8 @@ ExternalProject_Add(
     ${EXTERNAL_PROJECT_LOG_ARGS}
     PREFIX                ${MKLML_SOURCE_DIR}
     DOWNLOAD_DIR          ${MKLML_DOWNLOAD_DIR}
-    DOWNLOAD_COMMAND      wget --no-check-certificate -qO- ${MKLML_URL} | tar xz -C ${MKLML_DOWNLOAD_DIR}
+    DOWNLOAD_COMMAND      wget --no-check-certificate ${MKLML_URL} -c -O ${MKLML_VER}.tgz 
+                          && tar zxf ${MKLML_DOWNLOAD_DIR}/${MKLML_VER}.tgz
     DOWNLOAD_NO_PROGRESS  1
     UPDATE_COMMAND        ""
     CMAKE_ARGS            -DCMAKE_INSTALL_PREFIX=${MKLML_INSTALL_ROOT}

